.github/workflows/wipac-cicd.yml

```diff
@@ -118,6 +118,19 @@ jobs:
         run: |
           cat pytest.logs
 
+  test-build-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/setup-buildx-action@v2
+      - uses: docker/build-push-action@v3
+        with:
+          context: .
+          cache-from: type=gha
+          cache-to: type=gha,mode=min
```

Comment:
This Dockerfile won't make good use of the cache, since it copies the files in as one of the first actions (and that's not very cacheable).  Maybe best to remove these two lines.